service/src/main/java/uk/gov/hmcts/reform/fpl/service/validators/AllocationProposalChecker.java

```diff
@@ -21,6 +22,14 @@
     public boolean isStarted(CaseData caseData) {
         final Allocation allocationProposal = caseData.getAllocationProposal();
         return isNotEmpty(allocationProposal)
-                && anyNonEmpty(allocationProposal.getProposal(), allocationProposal.getProposalReason());
+            && anyNonEmpty(allocationProposal.getProposal(), allocationProposal.getProposalReason());
+    }
+
+    @Override
```

Comment:
worth following the same pattern as per AllocationProposalCheckerStartTest (or group the tests with nested) ? In a unit test normally you want to test 1 method only. That would be valid for all the other Checkers